src/sage/features/__init__.py

```diff
@@ -802,7 +808,7 @@ class StaticFile(FileFeature):
         To install no_such_file...you can try to run...sage -i some_spkg...
         Further installation instructions might be available at http://rand.om.
     """
-    def __init__(self, name, filename, *, search_path=None, **kwds):
+    def __init__(self, name, filename, search_path=None, type='optional', **kwds):
```

Comment:
the `*` there was there intentionally. It marks the following parameters as keyword-only (cannot be passed as positional)

Reply:
Thanks! I will do this later on.

Reply:
This id done, now! I took the occasion to also improve some doctest tags concerning the Gap package Grape which I came across.